src/redis_cmd.cc

```diff
@@ -820,12 +815,11 @@ class CommandDel : public Commander {
 };
 
 Status getBitOffsetFromArgument(std::string arg, uint32_t *offset) {
-  int64_t offset_arg = 0;
-  try {
-    offset_arg = std::stoll(arg);
-  } catch (std::exception &e) {
+  auto parseResult = ParseInt<int64_t>(arg, 10);
```

Comment:
uint33_t

Reply:
Hi. I think that in this simple pr, we do not need to modify the data type according to logic, and it is important to restore the original data type as much as possible. I think we can reopen a PR to make these logical adjustments. What do you think.

Reply:
If you parse a int64_t, and then check whether its range is exactly in (0, UINT32_MAX), you can just parse a uint32_t instead.

Reply:
BTW, you can just return the original status.
```
return res.ToStatus();
```

Reply:
> If you parse a int64_t, and then check whether its range is exactly in (0, UINT32_MAX), you can just parse a uint32_t instead.

yes I know what you mean. I mean, should this pr be defined as trying to restore the original code.

Reply:
That is to say, the type used in the original code shoule be the same here. In this pr, regardless of whether the type can be adjusted logically. If the type can be modified, we can do it in the next Pr?